src/handler/http/models/mod.rs

```diff
@@ -16,4 +16,7 @@
 //! This module contains models that can be serialized and deserialized as JSON
```

Comment:
Struct definitions here are all copy-pasted from `config::meta` since, prior to this PR, the JSON de/serialization logic in `config::meta` defines the HTTP API contracts.